gamemodes/terrortown/gamemode/client/cl_player_ext.lua

```diff
@@ -114,6 +114,91 @@ function plymeta:AnimUpdateGesture()
     end
 end
 
+---
+-- Generates a fake voice spectrum based on the player voice volume that looks like
+-- the restult of an FFT.
+-- @param[default=16] number stepCount Defines the result's resolution
+-- @return table Returns the table with the fake spectrum
+-- @realm client
+function plymeta:GetFakeVoiceSpectrum(stepCount)
```

Comment:
Can we move this somewhere else? I feel like this is a very specialized function that is not necessarily needed on a player directly and could rather take a player as a parameter. This would also reduce the amount of unrelated functions in this file.